.github/workflows/indexer-api-documentation-check.yml

```
@@ -19,11 +19,11 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)